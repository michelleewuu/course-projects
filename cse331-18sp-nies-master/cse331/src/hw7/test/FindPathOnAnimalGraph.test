# Tests find minimum-cost path on animal graph.

# Load animal.tsv
LoadGraph animal animal.tsv

# Test find minimum-cost path not shortest path
FindPath animal Penguin Monkey