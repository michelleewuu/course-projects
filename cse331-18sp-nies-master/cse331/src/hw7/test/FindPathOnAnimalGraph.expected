# Tests find minimum-cost path on animal graph.

# Load animal.tsv
loaded graph animal

# Test find minimum-cost path not shortest path
path from Penguin to Monkey:
Penguin to Otter with weight 0.333
Otter to Monkey with weight 0.333
total cost: 0.667